index.js

```diff
@@ -40,10 +46,14 @@ exports.run = function(runner, specs) {
         } catch (err) {
           reject(err);
         }
-      });
+      }
     });
   });
 
+  function isCucumber2() {
+    return !!Cucumber.defineSupportCode;
```

Comment:
Sure, but I think the cucumber team follows semantic versioning pretty well. If defineSupportCode was changed that would be a breaking change warranting a new major version number.